ueransim/etc/ueransim/open5gs-gnb.yaml

```diff
@@ -0,0 +1,23 @@
+mcc: "001"         # Mobile Country Code value
```

Comment:
remove open5gs name from open5gs-gnb.yaml configuration. We should not make references to a specific core in the ueransim image